server/aws-lsp-codewhisperer/src/language-server/languageDetection.ts

```diff
@@ -1,58 +1,96 @@
 import { TextDocument } from '@aws/language-server-runtimes/server-interface'
 
 export type CodewhispererLanguage =
-    | 'java'
-    | 'python'
-    | 'jsx'
-    | 'javascript'
-    | 'typescript'
-    | 'tsx'
-    | 'csharp'
     | 'c'
     | 'cpp'
-    | 'cpp'
+    | 'csharp'
     | 'go'
+    | 'java'
+    | 'javascript'
+    | 'jsx'
     | 'kotlin'
     | 'php'
+    | 'plaintext'
+    | 'python'
     | 'ruby'
     | 'rust'
     | 'scala'
     | 'shell'
-    | 'shell'
     | 'sql'
-    | 'plaintext'
+    | 'tsx'
+    | 'typescript'
+
+    // These work as well
```

Comment:
Should we just include these in the list above?

Reply:
Sure I will just include this in the list